cpp/tests/lists/explode_tests.cpp

```diff
@@ -547,6 +547,65 @@ TEST_F(ExplodeOuterTest, Nulls)
   CUDF_TEST_EXPECT_TABLES_EQUAL(pos_ret->view(), pos_expected);
 }
 
+TEST_F(ExplodeOuterTest, AllNulls)
+{
+  //    a            b
+  //    []           100
+  //    []           200
+  //    []           300
+
+  auto non_valid = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return false; });
+
+  LCW a({LCW{0}, LCW{0}, LCW{0}}, non_valid);
```

Comment:
It took me a while to understand why the column `a` contains all nulls but in the code it contains all `0`. I would prefer to use a sentinel variable like `constexpr auto null = 0;` and then put `null` everywhere, instead of using `0` assuming that it is null with a null mask. So we will have the code looks like `LCW a({LCW{null}, LCW{null}, LCW{null}}, non_valid);`, which is much more easier to understand.

Reply:
how about `LCW a({LCW{}, LCW{}, LCW{}}, non_valid);` ?

Reply:
That may work, but it is less intuitive 😄